app/assets/stylesheets/modules/_article_viewer.styl

```diff
@@ -76,14 +80,16 @@
 .user-legend-wrap
   width 90%
   background white
-
-.user-legend
+  
+.user-legend-article-viewer
```

Comment:
Is renaming this necessary? It seems like a pretty wordy class name.

Reply:
Okay. Maybe instead, make both of them more specific, for example by making it apply only to child components of `.article-viewer`? Or call it `article-viewer-legend` instead?

Ideally, we don't want any CSS in specific modules like this from affecting other modules, so adding more top-level rules is dangerous. I guess it was the recent change to the diff_viewer version that caused this problem?